utilities/virt.py

```diff
@@ -1020,7 +1020,7 @@ def _sc_name_for_storage_api():
 
     @property
     def virtctl_port_forward_cmd(self):
-        return f"{VIRTCTL} port-forward --stdio=true vm/{self.name}/{self.namespace} {SSH_PORT_22}"
+        return f"{VIRTCTL} port-forward --stdio=true vm/{self.name}.{self.namespace} {SSH_PORT_22}"
```

Comment:
should we add a TODO here or comment to fix / in 4.21 and later.

Reply:
it should be backlog jira task to track it and not TODO